Error when publishing a release to github

For some reason I started getting this error when publishing to GitHub, although it _seems_ as though the actual release works fine:

```
> search-index-adder@0.3.6 github-release /Users/fergusmcdowall/node/search-index-adder
> conventional-github-releaser -p angular

[ { url: 'https://api.github.com/repos/fergiemcdowall/search-index-adder/releases/6932093',
    assets_url: 'https://api.github.com/repos/fergiemcdowall/search-index-adder/releases/6932093/assets',
    upload_url: 'https://uploads.github.com/repos/fergiemcdowall/search-index-adder/releases/6932093/assets{?name,label}',
    html_url: 'https://github.com/fergiemcdowall/search-index-adder/releases/tag/v0.3.6',
    id: 6932093,
    tag_name: 'v0.3.6',
    target_commitish: 'master',
    name: 'v0.3.6',
    draft: false,
    author:
     { login: 'fergiemcdowall',
       id: 1122333,
       avatar_url: 'https://avatars2.githubusercontent.com/u/1122333?v=3',
       gravatar_id: '',
       url: 'https://api.github.com/users/fergiemcdowall',
       html_url: 'https://github.com/fergiemcdowall',
       followers_url: 'https://api.github.com/users/fergiemcdowall/followers',
       following_url: 'https://api.github.com/users/fergiemcdowall/following{/other_user}',
       gists_url: 'https://api.github.com/users/fergiemcdowall/gists{/gist_id}',
       starred_url: 'https://api.github.com/users/fergiemcdowall/starred{/owner}{/repo}',
       subscriptions_url: 'https://api.github.com/users/fergiemcdowall/subscriptions',
       organizations_url: 'https://api.github.com/users/fergiemcdowall/orgs',
       repos_url: 'https://api.github.com/users/fergiemcdowall/repos',
       events_url: 'https://api.github.com/users/fergiemcdowall/events{/privacy}',
       received_events_url: 'https://api.github.com/users/fergiemcdowall/received_events',
       type: 'User',
       site_admin: false },
    prerelease: false,
    created_at: '2017-07-03T05:13:19Z',
    published_at: '2017-07-05T05:56:41Z',
    assets: [],
    tarball_url: 'https://api.github.com/repos/fergiemcdowall/search-index-adder/tarball/v0.3.6',
    zipball_url: 'https://api.github.com/repos/fergiemcdowall/search-index-adder/zipball/v0.3.6',
    body: '### Features\n\n* **indexing:** can now do appendOnly ([be5167e](https://github.com/fergiemcdowall/search-index-adder/commit/be5167e))',
    meta:
     { 'x-ratelimit-limit': '5000',
       'x-ratelimit-remaining': '4997',
       'x-ratelimit-reset': '1499234292',
       'x-oauth-scopes': 'repo',
       location: 'https://api.github.com/repos/fergiemcdowall/search-index-adder/releases/6932093',
       etag: '"48c9e37b6d98af87149133b3ae640fe5"',
       status: '201 Created' } } ]

npm ERR! Darwin 16.6.0
npm ERR! argv "/Users/fergusmcdowall/.nvm/versions/node/v7.9.0/bin/node" "/Users/fergusmcdowall/.nvm/versions/node/v7.9.0/bin/npm" "run" "github-release"
npm ERR! node v7.9.0
npm ERR! npm  v4.2.0
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! search-index-adder@0.3.6 github-release: `conventional-github-releaser -p angular`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the search-index-adder@0.3.6 github-release script 'conventional-github-releaser -p angular'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the search-index-adder package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     conventional-github-releaser -p angular
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs search-index-adder
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls search-index-adder
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/fergusmcdowall/.npm/_logs/2017-07-05T05_56_41_987Z-debug.log
```

--- a/src/cli.js
+++ b/src/cli.js
@@ -115,19 +115,7 @@ conventionalGithubReleaser({
     process.exit(0);
   }
 
-  var allRejected = true;
-
-  for (var i = data.length - 1; i >= 0 ; i--) {
-    if (data[i].state === 'fulfilled') {
-      allRejected = false;
-      break;
-    }
-  }
-
-  if (allRejected) {
-    console.error(data);
-    process.exit(1);
-  } else if (flags.verbose) {
+  if (flags.verbose) {
     console.log(data);
   }
 });

